independent-publisher-connectors/Workable/Readme.md.txt

```diff
@@ -0,0 +1,38 @@
+### Workable Connector
```

Comment:
Hi @DavidBreathe thank you so much for your new submission.
May I ask you please to follow our template for the readme file [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md)
Please fix the file and include all required fields.

Reply:
Updated with new readme file from template